packages/alfa-rules/test/common/predicate/is-visible.spec.tsx

```diff
@@ -61,6 +61,37 @@ test(`isVisible() returns false when an element is hidden by reducing its size
   t.equal(isVisible(element), false);
 });
 
+test(`isVisible() returns false when an element is hidden by reducing its size
+      to 1x1 pixels and clipping overflow`, (t) => {
+  const element = (
+    <div style={{ width: "0px", height: "0px", overflow: "hidden" }}>
+      Hello World
+    </div>
+  );
+
+  t.equal(isVisible(element), false);
+});
+
+test(`isVisible() returns false when an element is scroll and its size is 0x0 px`, (t) => {
```

Comment:
```suggestion
test(`isVisible() returns false when an element scrolls its overflow and its size is reduced to 0 pixel, thus hidding the scrollbar`, (t) => {
```
(similar for the `auto` case)